File: Neetcode/ContainsDuplicate.cpp
class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {
        unordered_set<int> uset;
        for (int n: nums){
            if (uset.find(n) != uset.end()){
                return true;
            }
            uset.insert(n);
        }
        return false;
    }
};

File: Neetcode/ContainsDuplicate.py
class Solution:
    def containsDuplicate(self, nums: List[int]) -> bool:
        if len(nums) == len(set(nums)):
            return False
        else:
            return True
          
# class Solution:
#     def containsDuplicate(self, nums: List[int]) -> bool:
#         dct = {}
#         for n in nums:
#             if n in dct.keys():
#                 return True
#             else:
#                 dct[n] = True
#         return False
